docs/fundamentals/full-stack.md

```diff
@@ -1,10 +1,10 @@
 # Building Full-Stack Applications using Angular and NestJS
 
-In this guide we will build a full-stack application using Angular and NestJS.
+In this guide you will build a full-stack application using Angular and NestJS.
```

Comment:
I like the change from "we" to "you" for guides.

It gives more of an achievement for the reader.